src/content/datachannel/datatransfer/index.html

```diff
@@ -46,7 +46,7 @@
 
     <section>
       <div>
-        <input type="number" id="megsToSend" min="1" name="megs" value="1048"/>
+        <input type="number" id="megsToSend" min="1" name="megs" value="128"/> Megs
         <button id="sendTheData" type="button">Generate and send data</button>
```

Comment:
Suggest adding a checkbox for ordered/unordered. We don't have any unordered demos, and we had a crasher in this area recently. The rest of the demo should work fine as-is in either case.